### DISCUSSION TITLE
Plain text extraction methods

### DISCUSSION THREAD

**Discussion by prasantpoudel** [apify/crawlee-python#658] [Q&A]
As of now there is only title and content extraction methods are there(may other also vailable), also we can implement another methods that extract the plain text after removing all the tags from the html.

**Comment by janbuchar** [comment]
Hello, and thank you for your interest in Crawlee! Sophisticated text extraction is out of scope of this project. 

- If you just need to remove HTML tags completely, period, then you can code that in your request handlers using whatever facilities Playwright/Beautifulsoup/Parsel provides to you.
  - I have opened https://github.com/apify/crawlee-python/issues/659 for adding a simple helper that does this using BeautifulSoup. 
- If you're looking to extract **content** from the pages, you might want to use something like [crawl4ai](https://crawl4ai.com/mkdocs/). You can even integrate it with `BasicCrawler` from Crawlee.
- If you're looking for a complete text extraction solution, you can use the [Website Content Crawler](https://apify.com/apify/website-content-crawler) from Apify.
